src/shared/ui/image/utils/checkForSuitable.ts

```diff
@@ -0,0 +1,19 @@
+const checkForSuitable = (imgUrl: string, width: number = 430) => {
+  const extention = imgUrl.slice(imgUrl.lastIndexOf('.') + 1);
```

Comment:
```suggestion
  const imgExtention = imgUrl.slice(imgUrl.lastIndexOf('.') + 1);
```

Reply:
`imgExtenSion`*

Reply:
Thanks! I used different names in the new implementation)